### PR TITLE
Tidy up performance dashboards

### PR DESCRIPTION
**Pull request by paulrobertlloyd**
## Context

Copy and pasting code within dashboards has meant a few bad habits have been repeated. 

## Changes proposed in this pull request

* Remove repeated `headline-stat` `id` attributes (`id`s should be unique)
* Add useful page fragment identifiers
* Use `data-qa` for test hooks
* Ensure tile component always has the same bottom border (which means this doesn’t need to be set in every template)
* Update markup for dashboards (consistent spacing and heading sizes)

## Things to check

- [ ] This code does not rely on migrations in the same Pull Request
- [ ] If this code includes a migration adding or changing columns, it also backfills existing records for consistency
- [ ] API release notes have been updated if necessary
- [ ] Required environment variables have been updated [added to the Azure KeyVault](/docs/environment-variables.md#deploy-pipeline)

### CODE REVIEW

File: app/views/support_interface/performance/feature_metrics_dashboard.html.erb [hunk 1]
```diff
@@ -8,435 +8,433 @@
...
+            ) %>
+          </div>
         </div>
       </div>
     </div>
-  </div>
+  </section>
+
+  <section class="app-section">
+    <h2 id="satisfaction-survey" class="govuk-heading-m govuk-!-font-size-27">Satisfaction Survey</h2>
```
Comment (frankieroberto) [inline app/views/support_interface/performance/feature_metrics_dashboard.html.erb:380]:
```suggestion
    <h2 id="satisfaction-survey" class="govuk-heading-m govuk-!-font-size-27">Satisfaction survey</h2>
```

File: app/views/support_interface/performance/feature_metrics_dashboard.html.erb [hunk 2]
```diff
@@ -8,435 +8,433 @@
...
 
-  <h3 class="govuk-heading-m govuk-!-font-size-27 govuk-!-margin-top-8">Equality and Diversity</h3>
-  <div id="equality_and_diversity_dashboard_section" class="govuk-grid-row govuk-!-margin-bottom-4">
-    <div class="govuk-grid-column-one-half">
-      <div id="headline-stat" class="govuk-!-margin-bottom-4">
+    <div class="govuk-grid-row" data-qa="section-equality-and-diversity">
+      <div class="govuk-grid-column-one-half">
         <%= render SupportInterface::TileComponent.new(
           count: @dashboard.read_metric(:equality_and_diversity_response_rate),
           label: 'Equality and Diversity Survey Response Rate',
```
Comment (frankieroberto) [inline app/views/support_interface/performance/feature_metrics_dashboard.html.erb:417]:
```suggestion
          label: 'survey response rate',
```

**Review by frankieroberto — APPROVED** [review]
👏  nice work.

Can we try and use sentence case everywhere for headings too, and lowercase for the metric labels? I’ve left suggestions for the ones I spotted in the diff.

File: app/views/support_interface/performance/feature_metrics_dashboard.html.erb (hunk 1 above)
Comment (paulrobertlloyd) [reply app/views/support_interface/performance/feature_metrics_dashboard.html.erb]: Good spot. Fixed!

File: app/views/support_interface/performance/feature_metrics_dashboard.html.erb (hunk 2 above)
Comment (paulrobertlloyd) [reply app/views/support_interface/performance/feature_metrics_dashboard.html.erb]: Fixed!

**Pull request merged**